Signal/src/AppDelegate.m

```diff
@@ -417,7 +417,10 @@ - (void)applicationWillResignActive:(UIApplication *)application {
     dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
       if ([TSAccountManager isRegistered]) {
           dispatch_sync(dispatch_get_main_queue(), ^{
-              [self protectScreen];
+              if ([UIApplication sharedApplication].applicationState != UIApplicationStateActive) {
```

Comment:
It's part of the fix. We should never show the screen protection if the app is active.